pkgs/tools/misc/mongodb-compass/default.nix

```
@@ -42,14 +75,15 @@ let
     xorg.libXrandr
     xorg.libXrender
     xorg.libXtst
-    xorg.libXScrnSaver
+    xorg.libxkbfile
+    xorg.libxshmfence
   ] + ":${stdenv.cc.cc.lib}/lib64";
```

Does this work :thinking: ?

```suggestion
    xorg.libxshmfence
    (lib.getLib stdenv.cc.cc)
  ];
```

---

Yeap, it works :)